src/OrbitBase/include/OrbitBase/CrashHandlerBase.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_CRASH_HANDLER_BASE_H_
+#define ORBIT_BASE_CRASH_HANDLER_BASE_H_
+
+namespace orbit_base {
+
+class CrashHandlerBase {

```

Comment:
Could you add a small comment explaining that this is just a stub implementation and that the real implementation lives in CrashHandler::?

Reply:
Personal preference: I would call the class `CrashHandler`. The suffix `Base` sounds a bit like an implementation detail to me.

Reply:
Added a comment and updated the class name